### ISSUE TITLE
Example file not passing Core and unexpected passing for Tiles

### ISSUE THREAD

**Issue by bermud** [opengeospatial/ets-gpkg10#10]
When testing GeoPackage 1.0 revision 0.1 with this file:
http://www.geopackage.org/data/simple_sewer_features.gpkg

**Core** fails and **Tiles** is passing, even though it was not selected.

<img width="712" alt="geopk" src="https://cloud.githubusercontent.com/assets/3738898/17215327/84abf0ca-54aa-11e6-945e-44f54e21c8bc.png">

<img width="900" alt="core-gpk-failure" src="https://cloud.githubusercontent.com/assets/3738898/17215345/96ec83da-54aa-11e6-92ff-a0077e4c779b.png">

**Comment by bermud** [comment]
It was reported that running it via JUNIT test the file passes.

**Comment by bermud** [comment]
@keshav-nangare, please investigate why the test passes locally via JUnit and fails in beta?

**Comment by keshavnangare** [comment]
I have added the check for the conformance class "Tile".
- If the "Tile" is selected while executing the test, then it will execute both the conformance classes i.e.

```
     1.  Core 
     2.  Tile 
```
- If the "Core" is selected then it will execute only one conformance class i.e. `"Core"` and Tile class tests are skipped.

> Also I have updated the ConfigurationPolicy in the "testng.xml" file.

`<suite name="${ets-code}-${version}" verbose="0" configfailurepolicy="skip">`

Changed the value of configfailurepolicy i.e. `"continue" => "skip"`
